seed/urls/main.py

```diff
@@ -151,6 +153,43 @@
         name='delete_organization_buildings'
     ),
 
+    # timeseries data
```

Comment:
I have a hard time having helpful feedback on this because I don't have a lot of context for what this is doing or the functionality it is supporting.